### PR TITLE
Make sure virt-handler leaves VMs in final states alone

### PR DESCRIPTION
**Pull request by rmohr**

### CODE REVIEW

**Comment by rmohr** [comment]
retest this please

**Comment by mpolednik** [comment]
Roughly looks OK, one thing comes to mind - the code was catching too many events but there is no hint of that within the code itself. Point for #154 .

File: pkg/virt-handler/domain.go [hunk 1]
```diff
@@ -76,9 +80,11 @@ func (d *DomainDispatch) setVmPhaseForStatusReason(domain *virtwrap.Domain, vm *
 		switch domain.Status.Reason {
 		case virtwrap.ReasonCrashed:
 			vm.Status.Phase = v1.Failed
+			d.recorder.Event(vm, k8sv1.EventTypeWarning, v1.Stopped.String(), "The VM crashed.")
```
Comment (mpolednik) [inline pkg/virt-handler/domain.go]: Does v1.SyncEvent map to anything system related or is it just our construct? Does it make sense to have v1.Crashed?

Comment (rmohr) [inline pkg/virt-handler/domain.go]: Didn't want to introduce too much events, but might make sense.

**Comment by rmohr** [comment]
Verified on my system. There are new docker and kubeadm versions out, which introduce some issues.

**Pull request merged**